contracts/src/Vault.sol

```diff
@@ -375,6 +373,21 @@ contract Vault is Ownable2Step, IVault {
         }
     }
 
+    /// @notice Calculates the multiplier to be applied to the total supply to account for inflation
```

Comment:
I think we should change naming here, multiplier is strictly < SCALAR

youre returning inflation

Reply:
renamed to inflation